modules/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -488,17 +489,18 @@ private void writeBlobMultipart(BlobInfo blobInfo, byte[] buffer, int offset, in
     /**
      * Deletes the given path and all its children.
      *
+     * @param purpose
```

Comment:
I'd rather not have empty javadocs like this (ok to remove, no docs needed IMO, but if you want to keep it then please fill it in)

Reply:
This is an oversight. Thanks for catching it. The javadoc should be in place now.